docs/server/project_setup.rst

```diff
@@ -94,6 +94,13 @@ translation files for your new project. Make sure you have a
 Creating the project
 ++++++++++++++++++++
 
+.. note:: If you want to integrate your project with a version control
+   repository you need to :ref:`install the Pootle FS plugins for the VCS
+   <migrate_to_pootle_fs#install-vcs-plugins>` and :ref:`connect Pootle FS with
+   the VCS repository <migrate_to_pootle_fs#connect-with-vcs>` before creating
+   the project in Pootle.
```

Comment:
I like that VCS setup doesn't crowd this out, but again like I said earlier, I don't think this crucial info should be buried in migration.  It does need its own doc as it's a once off thing.